bin/client.js

```diff
@@ -264,16 +264,18 @@ let _check_for_post = ( is_post, options ) => {
 if ( !cline.args.length ||
      ( cline.args.length != 1 && cline.args.length != 2 ) ) { cline.help(); }
 
-var acceptTrailers = cline.acceptTrailers;
```

Comment:
Guess this changed from the previous versions of the library. In the docs they use const to catch opts
```
const opts = cline.opts();
```
Makes sense to put it in immutable if we are never going to change it. Could you please check if it is possible to use the `const` way? Just to keep it in line with the documentation of the library.